generate-key-pair-modal-component.js

```diff
@@ -5,27 +5,22 @@
  *
  * @author Dave Longley
  */
-define(['forge/js/pki'], function(pki) {
+import pki from 'forge/js/pki';
```

Comment:
Yes, the above (and extra line below) should work with forge 0.6.x.  0.7.x works differently and has some issues integrating with systemjs.  Eventually is will probably import all of forge and the extra line below will be removed.  And maybe in the far future forge will use es modules.